src/Extension/CommonMarkCoreExtension.php

```diff
@@ -64,6 +64,7 @@ public function register(ConfigurableEnvironmentInterface $environment)
             ->addInlineRenderer(InlineElement\HtmlInline::class, new InlineRenderer\HtmlInlineRenderer(), 0)
             ->addInlineRenderer(InlineElement\Image::class,      new InlineRenderer\ImageRenderer(),      0)
             ->addInlineRenderer(InlineElement\Link::class,       new InlineRenderer\LinkRenderer(),       0)
+            ->addInlineRenderer(InlineElement\Mention::class,    new InlineRenderer\LinkRenderer(),       0)
```

Comment:
Mentions are not part of the CommonMark spec so this shouldn't be added here.

However, because `Mention extends Link` and we implemented #222, this line may not be needed so long as we specify that our mention extension has a soft dependency on this extension.

Reply:
Ah, I forgot about #222. Yeah, that works.